pkg/e2e/options.go

```diff
@@ -17,8 +17,9 @@ limitations under the License.
 package e2e
 
 const (
-	etcdImage      = "quay.io/coreos/etcd:v3.0.14"
-	hyperkubeImage = "k8s.gcr.io/hyperkube:v1.5.1"
+	etcdImage       = "quay.io/coreos/etcd:v3.0.14"
+	hyperkubeImage  = "k8s.gcr.io/hyperkube:v1.5.1"
+	dnsmasqPtrImage = "gcr.io/kubernetes-e2e-test-images/dnsmasq-ptr:1.0"
```

Comment:
what is this?

Reply:
Now using k8s.gcr.io/k8s-dns-dnsmasq-amd64:1.14.5 instead.